tssh/login.go

```diff
@@ -1269,11 +1264,20 @@ func sshLogin(args *sshArgs) (ss *sshSession, err error) {
 	}
 
 	if !control {
-		// ssh agent forward
-		sshAgentForward(args, param, ss.client, ss.session)
+		// Let's postpone forwarding - maybe we won't have to do it.
+		afterLoginFuncs = append(afterLoginFuncs, func() {
+			// ssh forward
+			sshForward(ss.client, args, param)
```

Comment:
The warning message should be output before logging in, but now it may be output after logging in.